examples/mdl_sdk/dxr/mdl_d3d12/mdl_material_library.cpp

```diff
@@ -425,21 +422,21 @@ bool Mdl_material_library::reload_module(
         std::lock_guard<std::mutex> lock(m_targets_mtx);
         for (auto& hash : old_hashes)
         {
-            auto entry = m_target_map[hash];
-            if (!entry)
+            auto& entry = m_target_map[hash];
+            if (!entry.m_target)
```

Comment:
Previously is the !entry dose not make any sense, there is no way to store null pointer. 
As we are using value type for now, check the m_target instead.